src/semestr2homework1/services/PersonRunner.java

```diff
@@ -0,0 +1,67 @@
+package semestr2homework1.services;
+
+import semestr2homework1.models.Person;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class PersonRunner {
+    //Input example:
+    /*
+    Jack,Ward,male,16
+    Alex,Wilson,male,29
+    Monica,Lopez,female,33
+    Andrew,Warner,male,19
+    Anton,Waphphel,male,21
+     */
+    //Output example:
+    /*
+    Alex,Wilson,male,29
+    Andrew,Warner,male,19
+    Anton,Waphphel,male,21
+    Jack,Ward,male,16
+    Monica,Lopez,female,33
+     */
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String path;
+        System.out.println("Please, write path to input file:");
+        System.out.println("Example: C:/SomeDir");
+        //Path to file in my PC ---> C:/Users/Lenovo/IdeaProjects/TeachMeSkills/src/semestr2homework1/resources/input.txt
+        path = scanner.nextLine();
+        File personsFile = new File(path);
+        ArrayList<Person> personList = new ArrayList<Person>();
+        try (FileReader fr = new FileReader(personsFile); Scanner scan = new Scanner(fr)) {
+            while (scan.hasNextLine()) {
+                personList.add(buildPerson(scan.nextLine()));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
catch one exception and then throw another witout any additional logic is not good idea.
At least some sout to inform about the problem